tests/lib/smf/src/test_lib_flat_smf.c

```diff
@@ -244,11 +258,8 @@ void test_smf_flat(void)
 	test_obj.terminate = ENTRY;
 	smf_set_initial((struct smf_ctx *)&test_obj, &test_states[STATE_A]);
 
-	for (int i = 0; i < SMF_RUN; i++) {
-		if (smf_run_state((struct smf_ctx *)&test_obj)) {
-			break;
-		}
-	}
+	ret = smf_run_state((struct smf_ctx *)&test_obj);
+	zassert_equal(-1, ret, "Incorrect smf_run_state return value");
```

Comment:
Nit - the comment at 255 confused me at first as I thought the "B)" label was referring to a state name.  Can you remove all the labels?  `/* A) Test transitions */` becomes `/* Test transitions */`

Reply:
fixed